documentation/modules/auxiliary/scanner/http/kodi_traversal.md

```diff
@@ -0,0 +1,40 @@
+## Vulnerable Application
+
+This module exploits an arbitrary file disclosure vulnerability in Kodi 17.1.
+
+**Vulnerable Application Installation Steps**
+
+Grab whatever image from [libreelec](https://libreelec.tv/downloads/) if
```

Comment:
i get that libreelec is a kodi image essentially, but itd be nice to just have a quick thing here.  Right now it looks like we're exploiting kodi but then libreelec comes out of left field.

I'd also like to say a libreelec build here that is known vuln for future proofing.  in 5yrs this line wont change and someone will try a non-vuln version.